### DISCUSSION TITLE
Android Studio Build Gradle Failed

### DISCUSSION THREAD

**Discussion by yogiags86** [community/community#102811] [Programming Help]
### Body

Hi, I'm new here and just learning Android Studio. I tried building an APK from the Yuzu fork, but it displays an error message and Gradle build Failed. Do you know where the problem is?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by phuccoder2000** [comment]


**Reply by yogiags86** [reply]
Now the error configurecmakerelwithdebinfo[arm64-v8a] failed appears
